server/cli/src/cli.rs

```diff
@@ -515,6 +521,12 @@ async fn main() -> anyhow::Result<()> {
 
             info!("Report upserted");
         }
+        Action::ReloadEmbeddedReports => {
+            let connection_manager = get_storage_connection_manager(&settings.database);
+            let con = connection_manager.connection()?;
+
+            let _ = StandardReports::load_reports(&con, true);
```

Comment:
`let _` basically discards the result.

Should be something like `StandardReports::load_reports(&con, true)?` or `StandardReports::load_reports(&con, true).expect("Failed to load reports")`